content/doc/developer/publishing/releasing-experimental-updates.adoc

```diff
@@ -46,5 +53,8 @@ jenkins:
 
 === In official Docker images
 
-Official link:https://github.com/jenkinsci/docker[Jenkins Docker image] includes plugin management scripts which allow preinstalling plugins from the Experimental update center.
+The Official link:https://github.com/jenkinsci/docker[Jenkins Docker image] includes plugin management scripts which allow preinstalling plugins from the Experimental update center. 
+
+The container images support managing plugin versions as code so that the admministrator can define a container image that includes the exact plugin they require.
+
```

Comment:
```suggestion
The Official link:https://github.com/jenkinsci/docker[Jenkins Docker image] includes plugin management scripts, which allow preinstalling plugins from the experimental update center. 
The container images support managing plugin versions as code, so that the administrator can define a container image that includes the exact plugin they require.
```
suggestion for single sentence per line and some other syntax